tests/functional/artifacts/expected_manifest.py

```diff
@@ -31,6 +31,7 @@ def get_rendered_model_config(**updates):
         "meta": {},
         "unique_key": None,
         "grants": {},
+        "incremental_strategy": None,
```

Comment:
Should this be `None` by default, or should it be `"default"`? Any reason to prefer that logic in the adapter method (`get_incremental_strategy_macro`) versus here?

Reply:
If we make the default 'default' here it shows up in every node, even those that aren't incremental. That felt like a waste to me.  If we can come up with a way to make configs more pluggable it would be less of an issue.